main.go

```diff
@@ -22,7 +23,54 @@ const (
 	queryGetRooms = "SELECT * FROM rooms"
 )
 
+func isNewPlayer(r *http.Request) bool {
+	fmt.Printf("debug (isNewPlayer): number of cookies is %v\n", len(r.Cookies()))
+	return len(r.Cookies()) == 0
```

Comment:
this is not really check the thing we discussed yesterday, we need to check for the key of `player_id` exist and if exist if the value is `nil` or they value actually exist in the DB, right?